feature-libs/cart/import-export/components/import-to-cart/import-entries-dialog/import-entries-form/import-entries-form.component.html

```diff
@@ -7,12 +7,14 @@
       {{ 'importEntriesDialog.importProductFileDetails' | cxTranslate }}
     </p>
     <cx-file-upload
-      [control]="form.get('file')"
+      [formControl]="form.get('file')"
       [allowedExtensions]="fileValidity?.allowedExtensions"
-      (update)="loadFile($event, form)"
+      (update)="loadFile()"
     >
       {{ 'importEntriesDialog.selectFile' | cxTranslate }}
     </cx-file-upload>
+    <cx-form-errors [control]="form.get('file')" prefix="formErrors.file"></cx-form-errors>
```

Comment:
It's not the responsibility of the generic `<cx-file-upload>` component to display errors.
The responsibility of `<cx-file-upload>` is just to expose better value (FileList) than the native input type="file" (string file path).